index.hamlet

```diff
@@ -85,7 +85,7 @@ $doctype 5
       <thead>
         <tr>
           <th>Channel
-          <th>Last updated
+          <th>How old
```

Comment:
I feel like this should be more descriptive. Maybe `Time since last commit`?

Reply:
Reworded to `Time since last commit` and dropped the `old` from individual rows but it looks a little bit weird with a wide heading and fairly narrow contents.. Happy to hear any suggestions :).